crates/languages/src/rust.rs

```diff
@@ -85,7 +85,13 @@ impl LspAdapter for RustLspAdapter {
             "windows" => "pc-windows-msvc",
             other => bail!("Running on unsupported os: {other}"),
         };
-        let asset_name = format!("rust-analyzer-{}-{os}.gz", consts::ARCH);
+        let arch = consts::ARCH;
```

Comment:
Can we make it less runtime-dependent and do things in compile time, using existing constants?

https://github.com/zed-industries/zed/blob/9e7afe870a6f1e9a3a2debf65048a22ca0292271/crates/languages/src/typescript.rs#L297-L300

Reply:
Tried to get a bit closer to the typescript version, though can't quite use the utilities there. Lmk how this looks!